docs/.vitepress/config.mts

```diff
@@ -1,58 +1,65 @@
-import { defineConfig } from "vitepress";
-import { presetUno } from "unocss";
-import { presetRadix, radixColors } from "../../src";
-import unocss from "unocss/vite";
+import { defineConfig } from 'vitepress';
+import { presetUno } from 'unocss';
+import { presetRadix } from '../../src';
+import unocss from 'unocss/vite';
+import { RADIX_HUES } from '../../src/consts';
 
 // https://vitepress.dev/reference/site-config
 export default defineConfig({
-  title: "unocss-preset-radix",
-  base: "/unocss-preset-radix/",
+  title: 'unocss-preset-radix',
+  base: '/unocss-preset-radix/',
   vite: {
     plugins: [
       unocss({
         presets: [
           presetUno(),
           presetRadix({
-            palette: radixColors,
-            darkSelector: ".dark",
-
+            darkSelector: '.dark',
+            lightSelector: ':root, .light',
             aliases: {
-              testa: "red",
-              testb: "amber",
+              danger: 'red',
+              warning: 'amber',
+              accent: 'blue',
```

Comment:
This is a common thing, not just these lines: please configure whatever formatter you're using to not change every double quote in the project to a single quote, this is not the project style.